tests/test_functions.py

```diff
@@ -0,0 +1,119 @@
+import requests, difflib, sys
+import re
+
+command_line_arguments = []
+
+number_of_errors = 0
+
+
+def set_command_line_arguments(arguments):
```

Comment:
look into argparse for command line arg handling

Reply:
Good idea- realized I didn't need the global variable for command line arguments.